internal/appsec/listener/httpsec/http.go

```diff
@@ -113,8 +115,14 @@ func NewWAFEventListener(handle *waf.Handle, actions emitter.Actions, addresses
 				}
 			}
 		}
+		if canExtractSchemas(apiSecCfg) {
+			values["waf.context.processor"] = map[string]any{"extract-schema": true}
```

Comment:
So libddwaf saves this in its context and this is why you just need to pass it once?
If so, please add a note as a comment explaining it 🙏

Reply:
Indeed, which is verified in the tests where we check for tags generation at various steps (request start + body).
Comment: 50ef9aaf718ef